ci/ci.sh

```diff
@@ -302,10 +302,12 @@ test_cpp() {
     --test_arg=--ray_redis_password="1234"
   bazel test --test_output=all //cpp:test_python_call_cpp
 
-  # run the cpp example
-  rm -rf ray-template
-  ray cpp --generate-bazel-project-template-to ray-template
-  pushd ray-template && bash run.sh
+  # run the cpp example, currently does not work on mac
+  if [[ "${OSTYPE}" != darwin* ]]; then
+    rm -rf ray-template
+    ray cpp --generate-bazel-project-template-to ray-template
+    pushd ray-template && bash run.sh
```

Comment:
shouldn't we fix this rather than skipping?

maybe have two tests one with this, one without or something? so that the failure is separated rather than just skipped?

Reply:
let's create a ticket to fix it yes, my proposal is to stop running it on CI until that is fixed

Reply:
split into another job is also fine yes, but it burns money by keeping failing and retrying itself

Reply:
> burns money by keeping failing and retrying itself

they just `exit 1` here. I am okay with skipping running tests, but I prefer to have a red color if it should be release blocking.

Reply:
> red color if it should be release blocking

This is the problem yeah; the whole jobs are currently soft-failed and completely ignored (I removed the failing part and also removed the soft fail in this PR).